plugins/main/public/components/common/wazuh-discover/wz-discover.tsx

```diff
@@ -12,6 +12,7 @@ import {
   EuiFlyoutHeader,
   EuiTitle,
   EuiPanel,
+  EuiLink,
```

Comment:
If not used, it could be deleted

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6831/commits/42168192f7c168920b7b6cf111b812f32b894094